src/main/resources/CU-ApplicationResources.properties

```diff
@@ -738,3 +738,5 @@ errors.reject.invoice.po.vendor.mismatch=Vendor from Kuali PO does not match wit
 
 error.cam.building.room.combination.invalid=Invalid Room #%s for Building Code %s
 error.cam.building.code.invalid=Invalid Building Code %s
+
+error.uploadFile.extension=File '{0}' is not allowed as an attachment; please select a different file such as an Excel, Word or PDF document.
```

Comment:
Was this message provided by the functionals? If not, it would be good to confirm it with Cathy. Thank you.

Reply:
Cathy an I spoke about what the wording should be and she provided the specifics that are listed here.